ale/base/type_sensor.py

```diff
@@ -112,3 +113,72 @@ def ephemeris_stop_time(self):
           Ephemeris stop time for an image
         """
         return self.ephemeris_start_time + self.exposure_duration
+
+class Radar():
+    @property
+    def name_model(self):
+        """
+        Returns Key used to define the sensor type. Primarily
+        used for generating camera models.
+
+        Returns
+        -------
+        : str
+          USGS SAR (synthetic aperture radar) model
+        """
+        return "USGS_ASTRO_SAR_MODEL"
+
+    @property
+    def ephemeris_time(self):
+        newstart = spice.str2et(str(self.utc_start_time))
+        newend = spice.str2et(str(self.utc_stop_time))
```

Comment:
SPICE calls shouldn't be in the sensor type mix-in. These should only be in the SPICE mix-in or any driver the inherits from it.